datarepo-clienttests/src/main/java/scripts/testscripts/CreateSnapshot.java

```diff
@@ -64,8 +64,8 @@ public void setup(List<TestUserSpecification> testUsers) throws Exception {
     BulkLoadResultModel loadSummary = result.getLoadSummary();
     assertThat(
         "Number of successful files loaded should equal total files.",
-        loadSummary.getTotalFiles(),
-        equalTo(loadSummary.getSucceededFiles()));
+        loadSummary.getSucceededFiles(),
+        equalTo(loadSummary.getTotalFiles()));
```

Comment:
I just noticed why testing that the error message was backwards, so I went ahead and switched it!